scripts/copy-res.js

```diff
@@ -25,10 +25,13 @@ const INCLUDE_LANGS = [
     {'value': 'fi', 'label': 'Suomi'},
     {'value': 'fr', 'label': 'Français'},
     {'value': 'gl', 'label': 'Galego'},
+    {'value': 'hi', 'label': 'हिन्दी'},
     {'value': 'hu', 'label': 'Magyar'},
+    {'value': 'is', 'label': 'Íslensku'},
     {'value': 'it', 'label': 'Italiano'},
     {'value': 'ja', 'label': '日本語'},
     {'value': 'ko', 'label': '한국어'},
+    {'value': 'lt', 'label': 'Lietuvis'},
```

Comment:
What is your source for this spelling? I don't see it anywhere on https://en.wikipedia.org/wiki/Lithuanian_language at least...

Reply:
![image](https://user-images.githubusercontent.com/2403652/70043881-3032f380-15b9-11ea-895f-9b5684e26341.png)

Google Translate seems happy with it

Reply:
All of my sources are Google translate. In theory we should probably get the translators to fill this in.

I'll check against the ISO standard and see what it comes up with.

Reply:
I am concerned that that's the word for "a Lithuanian person" rather than the language, which Google Translate would have no way of distinguishing in the presented example.